skycoin/skywire
# Update to Angular 10

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

 Changes:	
- The Angular version used by the manager was updated from v9 to v10.

How to test this PR:
You should be able to continue using the manager normally, but before compiling it you will have to run `npm install` in `/static/skywire-manager-src` for updating the local copy of Angular and its dependencies.
